esx_service/utils/auth_data.py

```diff
@@ -0,0 +1,306 @@
+# VM based authorization for docker volumes
+
+import sqlite3
+import uuid
+import os
+
+class DbConnectionError(Exception):
+    """ An exception thrown when connection to a sqlite database fails """
+
+    def __init__(self, db_path):
+        self.db_path = db_path
+
+    def __str__(self):
+        return "DB connection error %s" % self.db_path
+
+class DockerVolumeTenant:
+
+    def __init__(self, name, description, default_datastore, default_privileges,
```

Comment:
For each tenant, they must specify a default_datastore and default_privileges.
The additional datastore/privileges is optional.